perma_web/perma/templates/docs/accounts.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base-responsive.html" %}
+{% block title %} | Docs{% endblock %}
+
+{% block meta_description %}
+This section of the user guide describes how Perma accounts work.
+{% endblock %}
+
+{% block mainContent %}
+
+<div class="container cont-fixed">
+  <div class="row">
+    <div class="col-sm-12">
+      <h1>Perma.cc user guide <span class="subsection">Accounts</span></h1>
+    </div>
+  </div>
+</div>
+
+{% include 'docs/docs_menu.html' %}
+
+<div class="container cont-reading cont-fixed" id="overview">
+  <div class="row">
+    <div class="col-sm-4 reading-toc" >
+      <ul>
+        <li class="reading-toc-chapter"><a href="#orgs">Accounts</a></li>
+        <li class="reading-toc-section"><a href="#who-needs-account">Who Needs an Account?</a></li>
+        <li class="reading-toc-section"><a href="#">Usage Limits</a></li>
+        <li class="reading-toc-chapter"><a href="#orgs">Organizations</a></li>
+        <li class="reading-toc-section"><a href="#free-service-academic">Free Service for Academic Organizations</a></li>
+        <li class="reading-toc-section"><a href="#free-service-courts">Free Service for Courts</a></li>
+        <li class="reading-toc-section"><a href="#paid-service">Paid Service for Other Organizations</a></li>
+        <li class="reading-toc-section"><a href="#authority">Authority and Responsibilities</a></li>
+        <li class="reading-toc-section"><a href="#adding-org-users">Adding and Removing Organization Users</a></li>
+      </ul>
+    </div>
+
+    <div class="col-sm-8 reading-body">
+      
+      <h2 class="body-ah">Accounts</h2>
+      <h3 id="who-needs-account" class="body-bh">Who Needs an Account?</h3>
+      <p class="body-text">To preserve web pages with Perma, you need an account. To register for a free, individual account, please visit our <a href="{% url 'sign_up' %}">signup page</a>.</p>
+      <p class="body-text">You don't need an account to view an archived page someone else made.</p>
+
+      <p></p>
+
+      <h3 id="becoming-org" class="body-bh">Usage Limits</h3>
+      <p class="body-text">Individual accounts let you preserve up to 10 web pages per month for free.</p>
+      <p class="body-text">To preserve more pages, you need to be affiliated with an organization that qualifies for free, unlimited service or one that has a paid subscription.</p>
+
+      <h2 class="body-ah">Organizational Accounts</h2>
```

Comment:
In the TOC, you like to this section using `<a href="#orgs">Organizations</a>`. So, that means this heading needs an id of "orgs". `<h2 id="orgs" class="body-ah">Organizational Accounts</h2>`